dandi/organize.py

```diff
@@ -256,6 +256,10 @@ def organize_external_files(
             ):
                 new_path = op.join(dandiset_path, op.dirname(e["dandi_path"]), name_new)
                 name_old_str = str(name_old)
+                if not op.isabs(name_old_str):
+                    name_old_str = op.join(op.dirname(e["path"]), name_old_str)
+                if not op.exists(name_old_str):
```

Comment:
Right now, if the file doesn't exist, you're just logging a message and then continuing as it if does.  Shouldn't you at least add a `continue`?

Reply:
I am just logging the error, but would want the`shutil.copy()` and `shutil.move()` or the symlink/hardlink to throw a `FileNotFoundError`

Reply:
why to hope/rely on some other code/error whenever you know that it must be coming?

Reply:
so we could explicitly throw a `FileNotFoundError` like https://github.com/dandi/dandi-cli/pull/931/commits/f23ca3677cc9b3b2259894ecf6440bad1f2fcc7e